contracts/token/ERC20/extensions/ERC20Freezable.sol

```diff
@@ -9,12 +9,12 @@ import {IERC7943Fungible} from "../../../interfaces/IERC7943.sol";
 /**
  * @dev Extension of {ERC20} that allows to implement a freezing
  * mechanism that can be managed by an authorized account with the
- * {_freezeTokens} and {_unfreezeTokens} functions.
+ * {_setFrozen} function.
  *
  * The freezing mechanism provides the guarantee to the contract owner
  * (e.g. a DAO or a well-configured multisig) that a specific amount
  * of tokens held by an account won't be transferable until those
- * tokens are unfrozen using {_unfreezeTokens}.
+ * tokens are unfrozen using {_setFrozen}.
```

Comment:
I think this would be clearer: `... won't be transferable until the frozen amount is reduced using {_setFrozen}.`